minimedia.cpp

```diff
@@ -74,11 +74,11 @@ main(int, char**)
     FakePermissionController::instantiate();
     FakeAppOps::instantiate();
     FakeBatteryStats::instantiate();
-#ifndef SENSORSERVER_DISABLE
+#if ! defined(SENSORSERVER_DISABLE) && ANDROID_MAJOR <= 9
```

Comment:
Nitpick: extra space
```suggestion
#if !defined(SENSORSERVER_DISABLE) && ANDROID_MAJOR <= 9
```